storybook/main.ts

```diff
@@ -47,4 +47,5 @@ module.exports = {
     "storybook-addon-designs",
   ],
   presets: [path.resolve("./storybook/header-preset/preset")],
+  staticDirs: ["./assets"],
```

Comment:
If we change this to:
 ```
  staticDirs: [{ from: "./assets", to: "/static/media" }],
```
then it will move those files to the static media folder at build 🕺

Reply:
still need to figure out the janky reference of [object Module] at build but figure we could do something like this [example](https://github.com/storybookjs/storybook/issues/9443#issuecomment-574713070) but inversed 🤷 
```
  brandImage: (process.env.NODE_ENV === 'production') ? logoUrl : '/logo.svg',`
// to
    brandImage: (process.env.NODE_ENV === 'production') ?  'static/media/logo.svg' : importedInSVG`